src/ai/backend/manager/models/vfolder.py

```diff
@@ -2222,6 +2251,7 @@ class Meta:
 
     name = graphene.String()
     vfolder = graphene.Field(VirtualFolder)
+    vfolder_node = graphene.Field(VirtualFolderNode, description="Added in 24.09.0.")
```

Comment:
If this PR will also be merged into 24.09.x, please update this description.

Reply:
`24.09.0` is the first minor version of 24.09. Or does any other version come to prior to 24.09.0?

Reply:
As we confirmed that the target milestone is `24.09`, this conversation has been resolved.